app/frontend/src/features/messages/Messages.tsx

```diff
@@ -1,16 +1,39 @@
 import { TabContext, TabPanel } from "@material-ui/lab";
-import * as React from "react";
+import React from "react";
 import PageTitle from "../../components/PageTitle";
 import GroupChatsTab from "./groupchats/GroupChatsTab";
 import SurfingTab from "./surfing/SurfingTab";
 import TabBar from "../../components/TabBar";
 import { useHistory, useParams } from "react-router-dom";
 import { messagesRoute } from "../../AppRoutes";
 
+import NotificationBadge from "../../components/NotificationBadge";
+import useNotifications from "../useNotifications";
+
+export function MessagesNotification() {
+  const { data } = useNotifications();
+
+  return (
+    <NotificationBadge count={data?.unseenMessageCount}>
+      Group Chats
+    </NotificationBadge>
+  );
+}
+
+export function CouchRequestsNotification() {
```

Comment:
For consistency with the api
```suggestion
export function HostRequestsNotification() {
```